sum/README.md

```diff
@@ -1,8 +1,11 @@
-This is "sum of 2 numbers" with assembly for Linux x86-64
+# Sum of numbers
 
-Build it with: `make`
+This is a basic program that gets the sum of two integer numbers and check that it is correct. It prints the message if the sum is correct, otherwise just exits.
```

Comment:
```suggestion
This is a basic program that sums two integer numbers and checks if it is correct. If the sum is correct, it prints the message; otherwise, it just exits.
```